sound/soc/sof/ops.c

```diff
@@ -163,7 +165,7 @@ int snd_sof_dsp_register_poll(struct snd_sof_dev *sdev, u32 bar, u32 offset,
 
 		/* Phase 2 after 5ms(500us * 10) */
 		if (++k > 10)
-			s = 5000;
+			s = delay_us*10;
```

Comment:
checkpatch issue here. needs spaces around the *

Reply:
I can change. BTW, I ran checkpatch.pl but there was no warning or error.

Reply:
@fredoh9 did you run it with "--notree --codespell --strict"?

Reply:
best thing would be to set up the pre and post-commit hooks in your linux repo just like the SOF repo